examples/IB/explicit/ex7/example.cpp

```diff
@@ -0,0 +1,435 @@
+            pout << "Simulation time is " << loop_time << "\n";
+            pout << "+++++++++++++++++++++++++++++++++++++++++++++++++++\n";
+            pout << "\n";
+
+            std::ofstream fout("Coordinates.txt", std::ios::app);
+
+            const int finest_ln = patch_hierarchy->getFinestLevelNumber();
+            Pointer<LData> X_data = ib_method_ops->getLDataManager()->getLData("X", finest_ln);
+            Vec X_vec = X_data->getVec();
+            double* x_vals;
+            int ierr = VecGetArray(X_vec, &x_vals);
+            IBTK_CHKERRQ(ierr);
+            Pointer<LMesh> l_mesh = ib_method_ops->getLDataManager()->getLMesh(finest_ln);
+            const std::vector<LNode*>& local_nodes = l_mesh->getLocalNodes();
+            for (const auto& node : local_nodes)
+            {
+                const int lag_idx = node->getLagrangianIndex();
+                const int petsc_idx = node->getLocalPETScIndex();
+                Eigen::Map<Vector3d> X(&x_vals[petsc_idx * NDIM]);
+                fout.unsetf(ios_base::showpos);
+                fout.setf(ios_base::scientific);
+                fout.precision(5);
+                fout << loop_time << "," << X(0) << "," << X(1) << "," << X(2) << "\n";
+            }
+
+            // At specified intervals, write visualization and restart files,
+            // print out timer data, and store hierarchy data for post
+            // processing.
+            iteration_num += 1;
+            const bool last_step = !time_integrator->stepsRemaining();
+            if (dump_viz_data && uses_visit && (iteration_num % viz_dump_interval == 0 || last_step))
+            {
+                pout << "\nWriting visualization files...\n\n";
+                time_integrator->setupPlotData();
+                visit_data_writer->writePlotData(patch_hierarchy, iteration_num, loop_time);
+                silo_data_writer->writePlotData(iteration_num, loop_time);
+            }
+            if (dump_restart_data && (iteration_num % restart_dump_interval == 0 || last_step))
+            {
+                pout << "\nWriting restart files...\n\n";
+                RestartManager::getManager()->writeRestartFile(restart_dump_dirname, iteration_num);
+            }
+            if (dump_timer_data && (iteration_num % timer_dump_interval == 0 || last_step))
+            {
+                pout << "\nWriting timer data...\n\n";
+                TimerManager::getManager()->print(plog);
+            }
+            if (dump_postproc_data && (iteration_num % postproc_data_dump_interval == 0 || last_step))
+            {
+            }
+        }
+
+        // Cleanup Eulerian boundary condition specification objects (when
+        // necessary).
+        for (unsigned int d = 0; d < NDIM; ++d) delete u_bc_coefs[d];
+
+    } // cleanup dynamically allocated objects prior to shutdown
+} // main
+
+void
+print_eul_data(Pointer<PatchHierarchy<NDIM> > hierarchy, LDataManager* l_data_manager)
```

Comment:
These are already printed to a different log file - we don't need to print them again. We should remove this function.